ingest/src/main/java/com/microsoft/azure/kusto/ingest/IngestClientImpl.java

```diff
@@ -172,7 +172,9 @@ public IngestionResult ingestFromStream(StreamSourceInfo streamSourceInfo, Inges
                 throw new IngestionClientException("Stream");
             }
             String blobName = genBlobName(
-                    "StreamUpload", ingestionProperties.getDatabaseName(), ingestionProperties.getTableName());
+                    "StreamUpload",
+                    ingestionProperties.getDatabaseName(),
+                    ingestionProperties.getTableName()) + ".gz";
```

Comment:
now that you have compression types in other PR, just use that instead of this string